event.c

```diff
@@ -120,6 +123,9 @@ static const struct eventop *eventops[] = {
 #ifdef EVENT__HAVE_SELECT
 	&selectops,
 #endif
+#ifdef EVENT__HAVE_WEPOLL
+	&wepollops,
+#endif
```

Comment:
If all the tests passes, maybe we should enable it by default? (since I don't see anything that will allow us to do so with some confident, hence why not now?)

Reply:
Ok, let's leave this as is, and once we will take a closer look (maybe some more benchmarks) we will make it by default (@ygj6 CC)

Reply:
As-is, the ordering prefers wepoll over win32. I would agree that the quickest way to gain confidence (outside of automated tests) is to just make it the default. I can change this PR to keep win32 the default if preferred. It seems there was some back and forth during this review :)

Reply:
Doh, mixed up selectops with win32ops, sorry.
Yes, let's move it to the end of the list, to avoid changing default backed